packages/ui/src/composites/wui-chip/index.ts

```diff
@@ -18,6 +18,10 @@ export class WuiChip extends LitElement {
 
   @property() public imageSrc?: string = undefined
 
+  @property() public imageIcon?: IconType = undefined
+
+  @property() public imageIconSize: SizeType = 'md'
```

Comment:
I added two properties so i can use icons as well. 

At the moment the `<wui-chip>` component only accepts `imageSrc` property which requires me to use `https://*` only